mobile/lib/features/brag/brag.dart

```diff
@@ -0,0 +1,426 @@
+                  ],
+                ),
+                Align(
+                    alignment: Alignment.topLeft,
+                    child: SizedBox(
+                      width: 55,
+                      height: 55,
+                      child: Padding(
+                          padding: const EdgeInsets.all(8.0),
+                          child: Container(
+                            decoration: BoxDecoration(
+                                color: Colors.white,
+                                border: Border.all(
+                                  color: tenTenOnePurple,
+                                ),
+                                borderRadius: const BorderRadius.all(Radius.circular(5))),
+                            child: Padding(
+                                padding: const EdgeInsets.all(2.0),
+                                child: SvgPicture.asset(
+                                  'assets/10101_logo.svg',
+                                )),
+                          )),
+                    )),
+                Align(
+                  alignment: Alignment.topRight,
+                  child: SizedBox(
+                      width: 55,
+                      height: 55,
+                      child: Padding(
+                        padding: const EdgeInsets.all(8.0),
+                        child: Container(
+                            decoration: BoxDecoration(
+                                color: Colors.white,
+                                border: Border.all(
+                                  color: tenTenOnePurple,
+                                ),
+                                borderRadius: const BorderRadius.all(Radius.circular(5))),
+                            child: Padding(
+                              padding: const EdgeInsets.all(2.0),
+                              child: SvgPicture.asset('assets/10101-qr-transparent.svg',
+                                  colorFilter:
+                                      const ColorFilter.mode(Colors.black, BlendMode.srcIn)),
+                            )),
+                      )),
+                ),
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.end,
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.only(top: 8.0, bottom: 8),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                        children: [
+                          Padding(
+                            padding: const EdgeInsets.only(left: 8),
+                            child: Row(
+                              children: [
+                                Text(
+                                  pnl.formatted(),
+                                  style: primaryTextStypeValue,
+                                ),
+                                Icon(
+                                  BitcoinIcons.satoshi_v2,
```

Comment:
🔧 I have to say I still hate the sats symbol lol. I would just say `sats` and avoid any possible confusion.